### ISSUE TITLE
Add aria roles across site templates

### ISSUE THREAD

**Issue by KuraFire** [KuraFire/goinglovefirst#4]
The website is made with the Royce jekyll template. This template currently lacks any aria roles beyond `aria-hidden="true"`. That's not great for accessibility, so let's improve it!

Implement a clear, well-design architecture of ARIA roles across the whole site:

1. [ ] Make proposed outline for how the roles should be used, and why
2. [ ] Check with a11y friends about the plan
3. [ ] Implement (with improvements from feedback)
